src/main/java/seedu/address/logic/commands/person/ViewGroupCommand.java

```diff
@@ -16,6 +16,7 @@
 public class ViewGroupCommand extends Command {
 
     public static final String COMMAND_WORD = "viewGroup";
+    public static final String COMMAND_WORD_LOWER = "viewgroup";
```

Comment:
I think we can just replace COMMAND_WORD with the lower case version instead of having 2 different constants for the command word

Reply:
but this will make the messages to the user use lower case version every time, I thought it's not that nice?

Reply:
Hmm would it be nicer for all uppercase? Alternatively we can just do 
switch (...):
    AddCommand.tolowercase():
.
.
.

in the AddressBookParser instead. I think it would be clearer since we dont need to deal with multiple similar constants.

Reply:
I think instead of having a second constant, maybe we can just do the .toLowerCase() part in the AddressBookParser